### ISSUE TITLE
OAuth 1.0a incorrect signature with URL-encoded parameters

### ISSUE THREAD

**Issue by osowskit** [a85/POSTMan-Chrome-Extension#8879]
**Describe the bug**
OAuth 1.0 Signature generation produces an incorrect hash when the URL params contain URL-encoded parameters.
`https://localhost:4567/fun?foo=first%2Csecond`. This is preventing Twitter developers from using Postman with the Twitter Ads API and was reported via the [forums](https://twittercommunity.com/t/multiple-metric-groups-fail-with-ads-api/138882/9)

**To Reproduce**

1. Create a request using OAuth Authentication Headers
   `HMAC-SHA1`, 
   **Consumer Key** `a`, 
   **Consumer Secret** `b`,
   **Access Token** `123`,
   **Token Secret** `abc`,
   **Timestamp** `123`,
   **Nonce** `3`
1. Add a GET request to `localhost:4567/fun?foo=first%2Csecond`
3. Inspect the Header for the Signature
```
OAuth oauth_consumer_key="a",oauth_token="123",oauth_signature_method="HMAC-SHA1",oauth_timestamp="123",oauth_nonce="3",oauth_version="1.0",oauth_signature="1tyKsJRLaTvpnCwKGayL4YgHmLM%3D"
```

**Expected behavior**
The generated `oauth_signature` should be `cdQQuXon1opN%2F3wj5APnIF5NAMg%3D`. The base string for the hash should be `GET&http%3A%2F%2Flocalhost%3A4567%2F&foo%3Dfirst%252Csecond%26oauth_consumer_key%3Da%26oauth_nonce%3D3%26oauth_signature_method%3DHMAC-SHA1%26oauth_timestamp%3D123%26oauth_token%3D123%26oauth_version%3D1.0`

Note that not using URL encoded parameters generate the correct signature - i.e. using `localhost:4567/fun?foo=first,second`. It would be helpful to debug this by inspecting the base string Postman generates for the HMAC.

**App information (please complete the following information):**
 - App Type [e.g. Chrome App, Native App]
 - Postman Version latest
 - OS: Catalina 10.15.6

**Comment by osowskit** [comment]
@vikiCoder let me know if you need any help reproducing this. As we've had a few Twitter developers run into this, we'd like to understand what's going on to help our community adopt our Postman collection.

**Comment by mccannt** [comment]
Thanks, @osowskit for the report. 
We are currently investigating this and will reach out if we have any further questions.

**Comment by osowskit** [comment]
We still get a handful of developers that run into this bug every month. Related to https://github.com/twitterdev/postman-twitter-ads-api/issues/2

**Comment by codenirvana** [comment]
@osowskit It took us a while but, we've finally fixed the URL encoding issue with OAuth 1.0 in Postman v8.7.0.
 Can you update your app and confirm if this fixes your issue?

**Comment by osowskit** [comment]
I was able to confirm this works correctly. Thanks for the help.
